events/guild/message.js

```diff
@@ -74,7 +74,12 @@ module.exports = async (message, client) => {
 	} else {
 		if (message.createdTimestamp - profileData.lastMessageTimestamp > ms("1w")) {
 			let days = Math.floor((message.createdTimestamp - profileData.lastMessageTimestamp) / 1000 / 86400);
-			profileData.xp -= days * 2;
+			let penalty = days * 2;
```

Comment:
Looks good! We should probably remove the old code as you said or make it reset the xp to 0 to save the hassle of manually resetting it. There are probably users right now who have negative xp values and when they try to type `.me` they will get an error, so I am voting for keeping the old code and resetting the xp to 0. Otherwise we will have to do it manually.

Reply:
Hm, I'm hesitant to do a fix like that since it means that we could be silencing a more severe issue in our codebase. I also looked up all users with XP<0 and there are less than 60 of them currently, so pesonally I want it to break so we can fix it if something is wrong instead of just resetting when something is wrong.